ignite/contrib/handlers/tensorboard_logger.py

```diff
@@ -405,7 +405,18 @@ def __init__(self, log_dir):
             raise RuntimeError("This contrib module requires tensorboardX to be installed. "
                                "Please install it with command: \n pip install tensorboardX")
 
-        self.writer = SummaryWriter(logdir=log_dir)
+        try:
+            self.writer = SummaryWriter(logdir=log_dir)
+        except TypeError as err:
+            if "type object got multiple values for keyword argument 'logdir'" == str(err):
+                self.writer = SummaryWriter(log_dir=log_dir)
+                from pkg_resources import get_distribution
```

Comment:
What do you think about removing `pkg_resources` (it seems like an additional?) and changing the warning message to
"tensorboardX version <1.7 will not be supported after ignite 0.3.0; please upgrade"